backend/src/main/resources/email_content/password_reset_ENG.txt

```diff
@@ -0,0 +1,8 @@
+Hello %s,
+
+You received  this message because you wanted to reset your password. Please click the link below to reset your password:
+
+http://localhost:3000/token/%s
```

Comment:
Może być to /reset-password/<token>? Bo nie będę wiedział czy użytkownik resetuje hasło, czy aktywuje konto.

Reply:
zmienione